sdk-setup/systemd/sdk-webappstub.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Nothing to see here go away
+
+[Service]
+Type=simple
+ExecStart=/bin/sh -c "nc -l -p 8080 -N <<<$'HTTP/1.0 404 Not Found\n\nThe SDK WebApp is gone, use sfdk\n\n'"
```

Comment:
As we are using bashism (<<<) here, I would be happier if we used /bin/bash instead of /bin/sh. Even though we now have /bin/sh pointing to /bin/bash, that's not something one should count on.

Reply:
And please use "Control Centre" instead of SDK WebApp, as that's the term that was used in the SDK IDE.

Reply:
And maybe "sfdk or Sailfish IDE"

Reply:
It seems that at least firefox refuses to show the content unless the Content-length header is there as well